.gitignore

```diff
@@ -6,7 +6,12 @@ package-lock.json
 __pycache__
 .*.swp
 .*.swo
+*.os
+*.o
+
+visionipc/test
 libcereal*.a
+libvisionipc.a
```

Comment:
should probably just ignore all *.a static libraries, don't think we have any checked in here